.eslintrc.json

```diff
@@ -1,25 +1,41 @@
+/*eslint import/no-cycle: [2, { ignoreExternal: true }]*/
+
 {
   "env": {
     "browser": true,
     "es6": true,
     "jest": true
   },
-  "parser": "babel-eslint",
   "parserOptions": {
     "ecmaFeatures": {
       "jsx": true
     },
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["airbnb", "plugin:react/recommended"],
-  "plugins": ["react"],
+  "extends": [
+    "airbnb",
+    "plugin:react/recommended"
+  ],
+  "plugins": [
+    "react"
+  ],
   "rules": {
-    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
-    "react/react-in-jsx-scope": "off",
+    "react/jsx-filename-extension": [
```

Comment:
- [ ] There is another rule `"react/react-in-jsx-scope": "off"` we need to add to comply with the [linters-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) provided by Microverse.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)